meson.build

```diff
@@ -45,6 +45,7 @@ udev           = dependency('libudev')
 pixman         = dependency('pixman-1')
 xcb            = dependency('xcb')
 xcb_composite  = dependency('xcb-composite')
+x11_xcb        = dependency('x11-xcb')
```

Comment:
Let's make this optional.

Reply:
Of course. We need to go through and make all of this X11 stuff optional.
I figured it would happen at the same time as Xwayland.

Reply:
Alright, can be done separately